diff --git a/externals/MQX_4_1/config/twrk70f120m/user_config.h b/externals/MQX_4_1/config/twrk70f120m/user_config.h
index 4040dd2..535b03a 100644
--- a/externals/MQX_4_1/config/twrk70f120m/user_config.h
+++ b/externals/MQX_4_1/config/twrk70f120m/user_config.h
@@ -38,12 +38,14 @@
  */
 #define BSPCFG_ENABLE_LEGACY_II2C_SLAVE     0
 
+#define MK70_REV_1_0             1
+
 /* MGCT: <generated_code> */
 #define BSPCFG_ENABLE_TTYA       0
 #define BSPCFG_ENABLE_ITTYA      0
 #define BSPCFG_ENABLE_TTYB       0
 #define BSPCFG_ENABLE_ITTYB      0
-#define BSPCFG_ENABLE_TTYC       1
+#define BSPCFG_ENABLE_TTYC       0
 #define BSPCFG_ENABLE_ITTYC      0
 #define BSPCFG_ENABLE_TTYD       0
 #define BSPCFG_ENABLE_ITTYD      0
@@ -51,24 +53,27 @@
 #define BSPCFG_ENABLE_ITTYE      0
 #define BSPCFG_ENABLE_TTYF       0
 #define BSPCFG_ENABLE_ITTYF      0
-#define BSPCFG_ENABLE_I2C0       1
-#define BSPCFG_ENABLE_II2C0      1
+#define BSPCFG_ENABLE_I2C0       0
+#define BSPCFG_ENABLE_II2C0      0
 #define BSPCFG_ENABLE_I2C1       0
 #define BSPCFG_ENABLE_II2C1      0
 #define BSPCFG_ENABLE_SPI0       0
 #define BSPCFG_ENABLE_SPI1       0
 #define BSPCFG_ENABLE_SPI2       0
-#define BSPCFG_ENABLE_RTCDEV     1
+#define BSPCFG_ENABLE_RTCDEV     0
 #define BSPCFG_ENABLE_PCFLASH    0
 #define BSPCFG_ENABLE_ADC0       0
-#define BSPCFG_ENABLE_ADC1       1
+#define BSPCFG_ENABLE_ADC1       0
 #define BSPCFG_ENABLE_ADC2       0
 #define BSPCFG_ENABLE_ADC3       0
 #define BSPCFG_ENABLE_FLASHX     0
 #define BSPCFG_ENABLE_ESDHC      1
-#define BSPCFG_ENABLE_IODEBUG    0
+
+#define BSP_DEFAULT_IO_CHANNEL "iodebug:"
+#define BSPCFG_ENABLE_IODEBUG    1
+
 #define BSPCFG_ENABLE_NANDFLASH  0
-#define BSPCFG_ENABLE_SAI        1
+#define BSPCFG_ENABLE_SAI        0
 
 #define MQX_USE_UNCACHED_MEM     1
 #define MQX_USE_LWMEM_ALLOCATOR  0
@@ -89,12 +94,12 @@
 #define FTPDCFG_USES_MFS         1
 #define RTCSCFG_ENABLE_SNMP      0
 
+#define RTCSCFG_ENABLE_DNS       1
+
 #define TELNETDCFG_NOWAIT        FALSE
 
 #define MQX_TASK_DESTRUCTION     1
 
-#define MQX_USE_TIMER            1
-
 /* MGCT: </generated_code> */
 
 #define ENETCFG_SUPPORT_PTP      0
@@ -102,8 +107,17 @@
 /*
 ** include common settings
 */
-#define MQX_USE_IO_OLD          1
 
+//------------------------------------------------
+//FIT overwrite default settings:
+#define MQX_USE_SEMAPHORES                  1
+#define MQX_KERNEL_LOGGING                  1
+#define MQX_EXTRA_TASK_STACK_ENABLE         1
+#define MQX_USE_TIMER                       1
+#define MQX_USE_SW_WATCHDOGS                1
+#define BSPCFG_ENABLE_DCU4                  0
+
+//------------------------------------------------
 
 /* use the rest of defaults from small-RAM-device profile */
 #include "small_ram_config.h"
